.idea/modules.xml

```diff
@@ -3,6 +3,7 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/potatopeeps.iml" filepath="$PROJECT_DIR$/potatopeeps.iml" />
+      <module fileurl="file://$PROJECT_DIR$/potatopeeps.iml" filepath="$PROJECT_DIR$/potatopeeps.iml" />
```

Comment:
What is the purpose of this?

Reply:
IntelliJ probably autogenerated this, didn't type it in manually.